src/vanilla/webpack.config.js

```diff
@@ -43,9 +44,7 @@ const commonConfig = {
   },
 
   plugins: [
-    new webpack.EnvironmentPlugin({
-      LIFF_ID: 'yourliffid'
-    }),
+    new Dotenv({ systemvars: true }),
```

Comment:
`systemvars` sets whether or not to read the server's environment variables.
https://github.com/mrsteele/dotenv-webpack#properties